pkg/noun/nock.c

```diff
@@ -554,7 +554,7 @@ _n_arg(c3_y cod_y)
       return sizeof(c3_s);
 
     case SWIP: case SWIN:
-      return sizeof(c3_l);
+      return sizeof(c3_l_tmp);
```

Comment:
The bytecode format is architecture-independent and should be pinned to a static 32-bit width.